lib/lunchbot/lunchbot_data/office_lunch_order.ex

```diff
@@ -16,4 +17,25 @@ defmodule Lunchbot.LunchbotData.OfficeLunchOrder do
     |> cast(attrs, [:day, :office_id])
     |> validate_required([:day, :office_id])
   end
+
+  def get_lunch_order_for_office(office \\ "Providence", date \\ Date.utc_today()) do
+    filter_lunch_orders_to_today_only = from(o in OfficeLunchOrder, where: o.day == ^date)
```

Comment:
What if we are ordering the day before? This is what I did for #32 to basically sort for the nearest upcoming or current day
```.ex
office_lunch_order_id_query =
      from olo in "office_lunch_orders",
        select: [olo.id, olo.menu_id],
        where: olo.day >= ^today and olo.office_id == ^office_id,
        order_by: [asc: :day],
        limit: 1
```

For #32, `office_id` is currently hardcoded and will be handled in a future ticket. When we reuse `shared-cart.ex` this will also be relevant

Reply:
> What if we are ordering the day before

Ahh yup, I had incorrectly assumed that the timestamp for the `OFFICE_LUNCH_ORDERS` table was for the day of the ordering, but it makes sense it would be for the day the order is to be placed.